src/ui/templates/partials/insurance/buyer-outstanding-payment-conditional-yes-html.njk

```diff
@@ -1,5 +1,19 @@
 {% import '../../components/monetary-value-input.njk' as monetaryValueInput %}
 
+{#
+  sets url for alternative currency link
+  if isChange - then adds /change to url
+  if isCheckAndChange - then adds /check-and-change to url
+  else sets it to PROVIDE_ALTERNATIVE_CURRENCY_URL
+#}
+{% if isChange === true %}
+  {% set alternativeCurrencyUrl = PROVIDE_ALTERNATIVE_CURRENCY_URL + "/change"%}
+{% elif isCheckAndChange === true %}
+  {% set alternativeCurrencyUrl = PROVIDE_ALTERNATIVE_CURRENCY_URL + "/check-and-change"%}
```

Comment:
Could we pass the URLs via the controller so the template just has e.g `CHANGE_URL`, `CHECK_AND_CHANGE_URL`? Like we do for the "save and back" URL